test.js

```diff
@@ -1,100 +1,41 @@
 var expect = require('chai').expect;
-var Bitmap_Transformer = require('./Bitmap_Transformer.js');
+var BitmapTransformer = require('./Bitmap_Transformer.js').BitmapTransformer;
 var fs = require('fs');
 
+var FitnessEmoji = new BitmapTransformer();
+
+
+describe('BitmapTransformer', function(){
+
+	it('has a read method', function (done){
```

Comment:
Technically because your read call is synchronous (uses `readFileSync`) you don't need to use mocha's `done` in the test function)